packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/apis/bulk_update.ts

```diff
@@ -273,14 +387,12 @@ export const performBulkUpdate = async <T>(
         return { type, id, error };
       }
 
-      // When a bulk update operation is completed, any fields specified in `_sourceIncludes` will be found in the "get" value of the
-      // returned object. We need to retrieve the `originId` if it exists so we can return it to the consumer.
-      const { _seq_no: seqNo, _primary_term: primaryTerm, get } = rawResponse;
+      const { _seq_no: seqNo, _primary_term: primaryTerm } = rawResponse;
 
       // eslint-disable-next-line @typescript-eslint/naming-convention
-      const { [type]: attributes, references, updated_at } = documentToSave;
+      const { [type]: attributes, references, updated_at } = documentToSave; // use the original request params ?? probably need to return the actual updated doc that exists in es now.
```

Comment:
I didn't want to change the signature of the API and optimistically assume that the updated documents are indexed.